plugins/callback/foreman.py

```diff
@@ -66,6 +66,13 @@
         ini:
           - section: callback_foreman
             key: verify_certs
+      disable:
+        description:
+          - Toggle to make the callback plugin disable itself even if it is loaded.
+          - It can be set to '1' to prevent the plugin from being used even if it gets loaded.
+        env:
+          - name: FOREMAN_DISABLE
```

Comment:
A thought:
Should this be more specific like `FOREMAN_CALLBACK_DISABLE`?
I'd expect this to work on all plugins not just callback if called so general, but that's probably not what we want here :thinking: